test/functional/feature_block.py

```diff
@@ -1125,7 +1128,6 @@ def run_test(self):
         tx1.vout.append(CTxOut(0, CScript([OP_TRUE])))
         tx1.vout.append(CTxOut(0, CScript([OP_TRUE])))
         tx1.vout.append(CTxOut(0, CScript([OP_TRUE])))
-        tx1.calc_sha256()
```

Comment:
Any reason to remove these? It seems safer to keep them just so the cached hash is up to date. (Future changes are less likely to break due to this, I'd say)

Reply:
The very next line of the code is `sign_tx()` which changes the signature data. And the line after it calls `rehash()` to forcibly calculate it again. So, this cached value was not being used. I don't mind adding it again